src/test/scala/chiselTests/ProbeSpec.scala

```diff
@@ -238,6 +239,7 @@ class ProbeSpec extends ChiselFlatSpec with Utils {
       "mismatched probe/non-probe types in ProbeSpec_Anon.io.out[0]: IO[Bool] and ProbeSpec_Anon.io.in[0]: IO[Bool]."
     )
   }
+  */
 
   ":= connector with probe/non-probe" should "fail" in {
     val exc = intercept[chisel3.ChiselException] {
```

Comment:
Does making a dedicated scala type help re:how these are printed? For example see, L258 where both probe and non-probe versions are just `IO[Bool]`.

Just a thought/a detail I keep meaning to investigate (probably could be fixed as-is?) :+1: .

Reply:
yes, it would. If i get to the point where these tests are actually running and failing would have to clean this up